src/parcsr_ls/HYPRE_parcsr_ls.h

```diff
@@ -741,14 +741,32 @@ HYPRE_Int HYPRE_BoomerAMGSetGridRelaxType(HYPRE_Solver  solver,
  *    - 4  : hybrid Gauss-Seidel or SOR, backward solve
  *    - 5  : hybrid chaotic Gauss-Seidel (works only with OpenMP)
  *    - 6  : hybrid symmetric Gauss-Seidel or SSOR
+ *    - 7  : Jacobi (uses Matvec)
  *    - 8  : \f$\ell_1\f$-scaled hybrid symmetric Gauss-Seidel
  *    - 9  : Gaussian elimination (only on coarsest level)
+ *    - 10 : On-processor direct forward solve for matrices with
+ *           triangular structure
+ *    - 11 : Two Stage approximation to GS. Uses the strict lower
+ *           part of the diagonal matrix
+ *    - 12 : Two Stage approximation to GS. Uses the strict lower
+ *           part of the diagonal matrix and a second iteration
+ *           for additional error approximation
  *    - 13 : \f$\ell_1\f$ Gauss-Seidel, forward solve
  *    - 14 : \f$\ell_1\f$ Gauss-Seidel, backward solve
  *    - 15 : CG (warning - not a fixed smoother - may require FGMRES)
  *    - 16 : Chebyshev
  *    - 17 : FCF-Jacobi
  *    - 18 : \f$\ell_1\f$-scaled jacobi
+ *    - 19 : Gaussian elimination (old version)
+ *    - 20 : Kaczmarz
+ *    - 21 : The same as 8 except forcing serialization on CPU (#OMP-thread = 1)
+ *    - 29 : Direct solve: use Gaussian elimination & BLAS
```

Comment:
I wasn't able to find 29 in the code.

Reply:
We have to be careful here:
Here are the definitions for the block relaxation schemes:
Switch statement to direct control based on relax_type:
    *     relax_type = 20 -> Jacobi or CF-Jacobi

    *     relax_type = 23 -> hybrid: SOR-J mix off-processor, SOR on-processor
    *                       with outer relaxation parameters (forward solve)
    *
    *     relax_type = 26 ->  hybrid: Jacobi off-processor,
    *                          Symm. Gauss-Seidel/ SSOR on-processor
    *                         with outer relaxation paramete
    *     relax_type = 29 -> Direct Solve

Reply:
so Kaczmarz should probably not be 20 either.

Reply:
Thank you @ulrikeyang . I guess this issue (Kaczmarz type 20) has been existing for a while. Do you think it is just type=20 that has the issue or anything >= 20?